_IBC/uptick-cosmoshub.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
```

Comment:
due to enforcement of alphabetical order, uptick should be chain 2 in both cases.

Reply:
Another PR has created these files for uptick